src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -185,8 +185,18 @@ public void postCommentPerIssue(String project, String repository, String pullRe
     }
   }
   
-  public StashCredentials getCredentials(){
-    return new StashCredentials(config.getStashLogin(), config.getStashPassword());
+  public StashCredentials getCredentials() throws StashConfigurationException {
```

Comment:
I wanted to have the user see if they asked for the environment variable usage and then fail to provide it.